web/src/engine/websites/Kakaopage.ts

```diff
@@ -0,0 +1,220 @@
+import { Tags } from '../Tags';
+import icon from './Kakaopage.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+type ApiClipBoard = {
+    contentHomeOverview: {
+        content: {
+            id: string,
+            seriesId: number,
+            title: string
+        }
+    }
+}
+
+type APIChapters = {
+    contentHomeProductList : {
+        edges: {
+            node: {
+                single: {
+                    productId: number,
+                    title: string
+                }
+            }
+        }[],
+        pageInfo: {
+            hasNextPage: boolean,
+            endCursor: string
+        }
+    }
+}
+
+type APiPages = {
+    viewerInfo: {
+        viewerData: {
+            imageDownloadData: {
+                files: {
+                    secureUrl: string
+                }[]
+            }
+        }
+    }
+}
+
+@Common.MangasNotSupported()
+@Common.ImageAjax()
+
+export default class extends DecoratableMangaScraper {
+    public constructor() {
+        super('kakaopage', `Page Kakao (카카오페이지)`, 'https://page.kakao.com', Tags.Media.Manga, Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Language.Korean, Tags.Source.Official);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return new RegExp(`^${this.URI.origin}/content/[\\d]+$`).test(url);
+    }
+
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
+        const id = parseInt(new URL(url).pathname.match(/content\/(\d+)/)[1]);
+        const gql = `
```

Comment:
Is GraphQL required, did `https://api2-page.kakao.com/api/v5` no longer work?

Reply:
IDK .I've fixed this website recently in Hakuneko and all i saw was graphql queries. 

![image](https://github.com/manga-download/haruneko/assets/10671518/49089015-551e-498a-8c91-eee13a29f068)

Reply:
I did not saw such call, and this host doesnt answer to pings for me.